doc/_templates/indexsidebar.html

```diff
@@ -18,6 +18,6 @@ <h3>{%trans%}Questions? Suggestions?{%endtrans%}</h3>
   <input type="submit" name="sub" value="Subscribe" />
 </form>
 </div>
-<p>{%trans%}or come to the <tt>#sphinx-doc</tt> channel on FreeNode.{%endtrans%}</p>
+<p>{%trans%}or come to the <tt>#sphinx-doc</tt> channel on <a href="https://web.libera.chat/?channel=#sphinx-doc">libera.chat</a>.{%endtrans%}</p>
```

Comment:
Is this migrated? I don't know who maintains this channel (I've never joined it!). So I can't evaluate this change is correct.

Reply:
#sphinx-doc on libera has now 7 users, the freenode one has 33. I posted a link to this PR about 4h ago but I got no replies yet. 

We will need to wait for more people to get confirmation on this move.

While 33 may seem a lot, I suspect most of them are not active and only lurking. On another project that I moved I had almost 50 lurkers.

Reply:
For the record, there are now 11 users on Libera. FreeNode has been actively banning bridges to Matrix and IRCCloud, so it's become a hostile place.